modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -176,7 +176,7 @@ export interface OfflineVaultTxInfo {
 export interface RecoverFromWrongChainOptions {
   txid: string;
   recoveryAddress: string;
-  wallet: Wallet;
+  wallet: string;
```

Comment:
this needs to be a string. in WRW they pass in a string, and thats how we do it in tests as well.